packages/slate/src/changes/by-key.js

```diff
@@ -281,6 +281,28 @@ Changes.removeMarkByKey = (change, key, offset, length, mark, options = {}) => {
 }
 
 /**
+ * Remove all `marks` from node by `key`.
+ *
+ * @param {Change} change
+ * @param {String} key
+ * @param {Object} options
+ *   @property {Boolean} normalize
+ */
+
+Changes.removeAllMarksByKey = (change, key, options = {}) => {
+  const { state } = change
+  const { document } = state
+  const node = document.getNode(key)
+  const texts = node.getTextsAsArray()
```

Comment:
I think this will fail if `node` is a text node, so we need to handle that case too

Reply:
fixed